org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/templates/XtextTemplateContext.java

```diff
@@ -88,37 +87,40 @@ protected TemplateTranslator createTemplateTranslator() {
 			int offset = getStart();
 			IRegion lineRegion = getDocument().getLineInformationOfOffset(offset);
 			String line = getDocument().get(lineRegion.getOffset(), lineRegion.getLength());
+			String lineDelimiter = getDocument().getLineDelimiter(0);
```

Comment:
There is an extension interface IDocumentExtension4 that offers `getDefaultLineDelimiter`. That's probably to be preferred.

Reply:
Done.